coreapps-web-2.2/pom.xml

```diff
@@ -0,0 +1,147 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+      <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>coreapps</artifactId>
+        <version>1.32.0-SNAPSHOT</version>
+     </parent>
+     <artifactId>coreapps-web-2.2</artifactId>
+     <packaging>jar</packaging>
+     <name>coreapps Module web 2.2</name>
+     <description>2.2 web project for coreapps</description>
+     
+     <properties>
+        <openMRSVersion>2.5.0-SNAPSHOT</openMRSVersion>
+        <reportingVersion>0.10.3</reportingVersion>
+        <calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
```

Comment:
Why not Core 2.x and non-snapshot?

And a more recent version of Reporting, and a non-snapshot of Calculation?

Reply:
hi @mks-d  when you talk about 2.x do you mean the recent version before snapshot?

Reply:
@mks-d   did you intend to mean **2.2.x**  ??

Reply:
> 2.x 

@mozzy11  i was referencing this